packages/account-kit/package.json

```diff
@@ -23,6 +23,10 @@
       ]
     }
   },
+  "bin": {
+    "aa-setup": "./dist/aa-setup.js",
+    "anvil-set-balance": "./dist/anvil-set-balance.js"
```

Comment:
I'm not really convinced this does enough to justify its own binary to maintain

Reply:
i feel like we need to provide something that's easier than telling people to copy paste `cast rpc --rpc-url http://127.0.0.1:8545 anvil_setBalance <address> $(cast to-unit 1ether wei)`

Reply:
I think this is fine, and gives folks exposure to cast + anvil RPC commands. My worry is that folks will start asking for more CLI options like allowing for setting RPC URL or whatever and then soon we'll just have a generic wrapper around `cast rpc anvil_setBalance`.

Reply:
discussed on discord: we'll remove this wrapper but will add a feature to the account kit bridge screen to fund the gas tank balance ("bridge from anvil to anvil") without executing a transaction from metamask